acid/features/history/tests/test_model.py

```diff
@@ -84,6 +85,23 @@ def test_build_number_return_none_if_no_build_number(self, make_buildset):
         buildset = make_buildset(build_number=None)
         assert buildset.build_number is None
 
+    @pytest.mark.parametrize("log_url, expected", [
+        ("http://logs.contrail.juniper.net/p/g/b/_/2019-04-15_57c498f/j/",
+            None),
+        ("http://logs.contrail.juniper.net/p/g/b/_/664/2019-04-15_57c498f/j/",
+            664),
+        ("http://logs.opencontrail.org/p/g/b/_/j/",
+            None),
+        ("http://logs.opencontrail.org/p/g/b/664/j/",
+            664)
+    ])
+    @db_session
+    def test_build_number_parsing(self, log_url, expected):
+        build = ZuulBuild(log_url=log_url)
+        assert build.build_number == expected
+
+    # TODO: analyze
```

Comment:
Do not smuggle changes irrelevant to the bugfix in this commit.
Separate this into a different commit. It can still be part of the pull request but as a different commit.

Reply:
Maybe it's not necessary when we use squash&merge here, but nice to note this in commit message.

Reply:
Kamil, I've split skipping the test to another commit. Your approval is crucial due to this project's settings (2 approvals out of 4 members needed).